ste/mgr-JobPartTransferMgr.go

```diff
@@ -694,16 +711,18 @@ const (
 
 func (jptm *jobPartTransferMgr) LogAtLevelForCurrentTransfer(level pipeline.LogLevel, msg string) {
 	// order of log elements here is mirrored, with some more added, in logTransferError
-	fullMsg := common.URLStringExtension(jptm.Info().Source).RedactSecretQueryParamForLogging() + " " +
+	info := jptm.Info()
+	fullMsg := common.URLStringExtension(info.Source).RedactSecretQueryParamForLogging() + " " + info.entityTypeLogIndicator() +
 		msg +
-		" Dst: " + common.URLStringExtension(jptm.Info().Destination).RedactSecretQueryParamForLogging()
+		" Dst: " + common.URLStringExtension(info.Destination).RedactSecretQueryParamForLogging()
 
 	jptm.Log(level, fullMsg)
 }
 
 func (jptm *jobPartTransferMgr) logTransferError(errorCode transferErrorCode, source, destination, errorMsg string, status int) {
 	// order of log elements here is mirrored, in subset, in LogForCurrentTransfer
-	msg := fmt.Sprintf("%v: ", errorCode) + common.URLStringExtension(source).RedactSecretQueryParamForLogging() +
+	info := jptm.Info() // TODO we are getting a lot of Info calls and its (presumably) not well-optimized.  Profile that?
```

Comment:
>// TODO we are getting a lot of Info calls and its (presumably) not well-optimized.  Profile that? [](start = 21, length = 98)

Not a bad idea. I wonder if we could get away with just memoizing it without too large of a hit on memory usage..

Reply:
Is on the backlog